app/ledger.js

```diff
@@ -467,16 +465,19 @@ var updatePublisherInfo = () => {
   syncWriter(pathName(publisherPath), data, () => {})
   syncWriter(pathName(scoresPath), synopsis.allN(), () => {})
 
+  if (synopsis.options.minDuration === 0) synopsis.options.minDuration = 8 * msecs.second
```

Comment:
because that is the actual default value, and when the synopsis is written out, i don't want it to be zero, so that when someone starts the browser without `process.env.NODE_ENV` being test, they won't get surprised... but, i see an easier fix. please stand by.